meta-lmp-bsp/recipes-security/optee/optee-os-fio-mfgtool_3.12.0.bbappend

```diff
@@ -0,0 +1,37 @@
+OPTEEMACHINE_apalis-imx6 = "imx-mx6qapalis"
+OPTEEMACHINE_qemuarm64 = "vexpress-qemu_armv8a"
+OPTEEMACHINE_imx6ullevk = "imx-mx6ullevk"
+OPTEEMACHINE_imx8mqevk = "imx-mx8mqevk"
+OPTEEMACHINE_imx8mmevk = "imx-mx8mmevk"
+OPTEEMACHINE_uz = "zynqmp-zcu102"
+
+EXTRA_OEMAKE_append_qemuarm64 = " \
+    CFG_RPMB_FS=y CFG_RPMB_WRITE_KEY=y \
+"
+EXTRA_OEMAKE_append_imx = " \
+    CFG_CAAM_DBG=0x001 \
+"
```

Comment:
On 3.10 we had CFG_NXP_WORKAROUND_CAAM_LOCKED_BY_HAB, did we forward ported the same change?

Reply:
CFG_OVERLAY_RESERVED_MEMORY_ADDRESS_CELLS=1 CFG_OVERLAY_RESERVED_MEMORY_SIZE_CELLS=1 \

I dont think we need these anymore